assets/_scss/elements/_inputs.scss

```diff
@@ -57,4 +57,94 @@ label {
 
 textarea {
   height: 16rem;
+}
+
+label.checkbox,
+label.radio {
+  display: inline-block;
+  font-weight: 400;
+  margin-bottom: 0.5em;
+
+  input[type="checkbox"] + span {
+    box-shadow: 0 0 0 1px $color-grey-600;
+    border-radius: $usa-border-radius;
+    display: inline-block;
+    height: 1.8rem;
+    margin-top: 0;
+    margin-right: 0.5em;
+    margin-bottom: 0;
+    vertical-align: -4px;
+    padding-top: 4px; //compensate for FA vertical-align
+    width: 1.8rem;
+  }
+
+  input[type="radio"] + span {
```

Comment:
Good point. @meiqimichelle wrote it this way so I wonder if using something else, like a class, would be more robust.